core/thread/CMakeLists.txt

```diff
@@ -27,6 +27,8 @@ else()
                          TWin32Thread.cxx TWin32ThreadFactory.cxx)
 endif()
 
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/../foundation/res)
```

Comment:
Hi Philippe, why is this needed? Also, why a relative directory and not `core/foundation/...`?